eno-core/src/main/java/fr/insee/eno/core/utils/vtl/VtlSyntaxUtils.java

```diff
@@ -11,85 +11,129 @@
 import java.util.List;
 
 /**
- * Utility class that provide methods for writing VTL expressions.
+ * Utility class that provide methods for analyzing/writing VTL expressions.
  */
 public class VtlSyntaxUtils {
 
-    // See: http://www.trevas.info/fr/user-guide/coverage/aggregate-operators
+    private VtlSyntaxUtils() {}
+
+    // ----- VTL syntax elements used in Eno
+
+    public static final String LEFT_JOIN_OPERATOR = getVTLTokenName(VtlTokens.LEFT_JOIN);
+    public static final String USING_KEYWORD = getVTLTokenName(VtlTokens.USING);
+
+    // ----- VTL syntax methods used in Eno
+
+    /**
+     * Returns a VTL expression that concatenates both VTL string expressions given.
+     * @param vtlString1 A VTL expression that returns a string value.
+     * @param vtlString2 A VTL expression that returns a string value.
+     * @return A VTL expression that concatenates both VTL string expressions given.
+     */
+    public static String concatenateStrings(String vtlString1, String vtlString2) {
+        return vtlString1 + " " + VtlSyntaxUtils.getVTLTokenName(VtlTokens.CONCAT) + " " + vtlString2;
```

Comment:
On peut retirer `VtlSyntaxUtils`
```suggestion
        return vtlString1 + " " + getVTLTokenName(VtlTokens.CONCAT) + " " + vtlString2;
```